DirectProgramming/DPC++/N-BodyMethods/Nbody/src/GSimulation.cpp

```diff
@@ -0,0 +1,230 @@
+//==============================================================
```

Comment:
Was this put through clang format?

Reply:
Yes, This was out through clang format with Google style.